include/deal.II/fe/fe_point_evaluation.h

```diff
@@ -211,6 +211,104 @@ namespace internal
 } // namespace internal
 
 
+/**
+ * @brief The namespace for the PointEvaluationFlags enum
+ *
+ * This namespace contains the enum PointEvaluationFlags used in
+ * FEPointEvaluation to control evaluation and integration of values, gradients,
+ * etc..
+ */
+namespace PointEvaluationFlags
+{
+  /**
+   * @brief The PointEvaluationFlags enum
+   *
+   * This enum contains a set of flags used by FEPointEvaluation::integrate(),
+   * FEPointEvaluation::evaluate() and others to determine if values and
+   * gradients or a combination of them is being used.
+   */
+  enum PointEvaluationFlags
```

Comment:
Shouldn't we try to use the same interface as for `FEEvaluation`? That is also added for this release, so it might make sense to add a single interface for evaluators (that is some smaller set of the `UpdateFlags`), even though we might have to assert some unsupported case in one or the other function.